docs/user-guide.md

```diff
@@ -227,6 +227,10 @@ Liberty Tools editing assistance provides code completion, diagnostics, and quic
 
 ![Eclipse LSP4MP microprofile-config.properties completion](/docs/user-guide-screenshots/mp_completion.png)
 
+3. To use Jakarta EE-specific and MicroProfile-specific quick-fixes, hover over a supported Jakarta EE or MicroProfile diagnostic. A pop-up dialog containing the diagnostic message and a quick-fix (code action) appears.
```

Comment:
```suggestion
3. To use Jakarta EE-specific and MicroProfile-specific quick-fixes, hover over a supported Jakarta EE or MicroProfile diagnostic. A pop-up dialog that contains the diagnostic message and  a **Quick Fix** link appears. Click the **Quick Fix** link to view a list of quick-fix options to choose from.
```

Reply:
Or, if Quick Fix does not appear in the UI:

```suggestion
3. To use Jakarta EE-specific and MicroProfile-specific quick-fixes, hover over a supported Jakarta EE or MicroProfile diagnostic. A pop-up dialog that contains the diagnostic message and  a quick fix link appears. Click this link to view a list of quick-fix options to choose from.
```

Reply:
@dmuelle thanks. I think we need to mention "if the quick fix is available". How about this?

To use Jakarta EE-specific and MicroProfile-specific quick-fixes, hover over a supported Jakarta EE or MicroProfile diagnostic. A pop-up dialog will appear that contains the diagnostic message and a "Quick Fix" link, if a quick-fix is available. Click the **Quick Fix** link to view a list of quick-fix options to choose from.

Reply:
Please check the latest.